text/0000-marketing-releases.md

```diff
@@ -0,0 +1,58 @@
+- Start Date: 2017-10-09
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Summary
+
+Add "named releases" to allow for easier marketing efforts while simultaneously maintaining Ember's strong semver guarantees.
+
+# Motivation
+
+Ember's commitment to semver is valuable. The practice of introducing new features in minor versions and major versions only removing deprecated functionality makes for a straightforward and predictable upgrade path.
```

Comment:
maybe spell out, link to the semver website, and then `(semver)` and use the shorthand for the rest of the RFC?